bindata/manifests/webhook/004-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
a comment on the name of the file.

should it be `004-configmaps.yaml` ?  or `004-nri-control-switches.yaml` ?

there may be other configmap that need to be deployed in the future from nri.

Reply:
@adrianchiris I think it shall be `004-configmap.yaml`. we have other manifests that are named after the singular kubernetes resource such as rbac, service or webhook etc. Example like webhook contains both mutating and validating webhooks, but the name is singular.

Reply:
ack, makes sense. additional configmaps would be added to this file.